native/swift/Tests/wordpress-api/Support/Extensions.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import WordPressAPI
+
+extension WpNetworkHeaderMap {
+    static var empty: WpNetworkHeaderMap {
+        // swiftlint:disable:next force_try
+        try! WpNetworkHeaderMap.fromMap(hashMap: [:])
+    }
+
+    static func withLinkHeader(_ value: String) -> WpNetworkHeaderMap {
+        // swiftlint:disable:next force_try
+        try! WpNetworkHeaderMap.fromMap(hashMap: ["Link": value])
+    }
+}
+
+// This might not be safe – we should figure that out
```

Comment:
These are children of the `HttpStubs`, so they need to be `Sendable`. They're not safe to be marked as such though because they're `uniffi::Object`-attributed, so we'll just do it in the test suite for now because we own that and can guarantee how it'll be used.